lldb/source/Plugins/Process/elf-core/ProcessElfCore.h

```diff
@@ -117,6 +117,7 @@ class ProcessElfCore : public lldb_private::PostMortemProcess {
     lldb::addr_t end;
     lldb::addr_t file_ofs;
     std::string path;
+    lldb_private::UUID uuid; //.note.gnu.build-id
```

Comment:
Add a comment letting the user know the UUID isn't actually in the NT_FILE_Entry:
```
// Add a UUID member for convenient access. The UUID value is not in the NT_FILE entries,
// we will find it in core memory and store it here for easy access.
```